esx_service/cli/vmdkops_admin.py

```diff
@@ -563,11 +563,14 @@ def get_creation_info(metadata):
 
 
 def get_attached_to(metadata):
-    """ Return which VM a volume is attached to based on its metadata """
+    """ Return which VM a volume is attached to based on its metadata. """
     try:
         vm_name = vmdk_ops.vm_uuid2name(metadata[kv.ATTACHED_VM_UUID])
         if not vm_name:
+            if metadata[kv.ATTACHED_VM_UUID]:
+                return metadata[kv.ATTACHED_VM_UUID]
```

Comment:
On second thought, I propose we bring vm_name back in KV. It may be stale in rare cases when vm_name is changed.

if (vm_name = vm_uuid2name(...))
    return vm_name
if (metadata[kv.ATTACHED_VM_NAME])
   vm_name = vm_name + "*"
   return vm_name
if (metadata[kv.ATTACHED_VM_UUID])
   return metadata[kv.ATTACHED_VM_UUID]

Log message in vmdkops_admin that 
`* Volume attached to VM running on another ESX Host.`

Other things we could do;
- Include ESX Host Name in KV.
- Always log both VM Name and UUID.

CC /@msterin